citysync/app/api/views.py

```diff
@@ -17,14 +17,11 @@ def jsonsift(obj, attrlist):
 
 @api_blueprint.route("/api/orgs/all", methods=["GET"])
 def get_orgs():
-    all_orgs = Nonprofit.query.all()
-    # paginate logic
-    records = []
-    for x, org in enumerate(all_orgs):
-        records.append(org)
-        if x == 10:
-            break
-
+    page = int(request.args.get('page', None))
+    if page is None:
+        records = Nonprofit.query.all()
+    else:
+        records = Nonprofit.query.paginate(page, 10, False).items
```

Comment:
As this stands. This does work to get us the first 10 records. However, This will only fetch the first 10 records regardless of the query parameter page value sent!

See Below:
# Specified with page 1
curl -X GET http://159.89.91.185:5050/api/orgs/all?page=1
# Specified with page 200!
curl -X GET http://159.89.91.185:5050/api/orgs/all?page=200

Since the things that we have to paginate thru are the records results. So we should try to fix that if possible in the paginate function. Or alternatively, if not possible we should be able to use direct SQL functions that mimic a `start` and `limit` queryparams typically seen in arrangements like this